run_bigquery.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
```

Comment:
I think we already have a [run_bigquery.sh](https://github.com/thiagotnunes/change-stream-sample/blob/main/run_bigquery.sh)

Reply:
Sorry, rebased.